src/datalink/dummy.rs

```diff
@@ -198,28 +178,19 @@ pub fn dummy_interface(i: u8) -> NetworkInterface {
 
 #[cfg(test)]
 mod tests {
-    use datalink::{EthernetDataLinkReceiver, EthernetDataLinkSender};
+    use datalink::{DataLinkReceiver, DataLinkSender};
     use datalink::Channel::Ethernet;
 
-    use packet::{MutablePacket, Packet};
-    use packet::ethernet::{EthernetPacket, MutableEthernetPacket};
     use std::io;
     use std::sync::mpsc::{self, Receiver, Sender, TryRecvError};
     use std::thread::{sleep, spawn};
     use std::time::Duration;
 
     #[test]
-    fn send_too_small_packet_size() {
-        let (_, _, mut tx, _) = create_net();
-        // Check that it fails to send with too small packet sizes
-        assert!(tx.build_and_send(1, 0, &mut |_| {}).is_none());
-    }
```

Comment:
Did you mean to remove this test?

Reply:
Yes. Now it's possible to send packets of any size. They are not longer constrained by having to be a size accepted by `EthernetPacket`